src/fvm_multicell.hpp

```diff
@@ -1075,13 +1087,25 @@ void fvm_multicell<Backend>::reset() {
     set_time_global(0);
     set_time_to_global(0);
 
+    // Update ion species:
+    //   - clear currents
+    //   - reset concentrations to defaults
+    //   - recalculate reversal potentials
```

Comment:
`ion::reset()` as defined below doesn't actually recalculate the reversal potentials (stale comment?)

Reply:
`ion::reset()` should calculate the reversal potentials, because they depend on the concentration values that are reset. I have fixed its implementation to do this.